confidant/routes/v1.py

```diff
@@ -780,6 +794,10 @@ def update_credential(id):
     credential_pairs = cipher.encrypt(update['credential_pairs'])
     update['metadata'] = data.get('metadata', _cred.metadata)
     update['documentation'] = data.get('documentation', _cred.documentation)
+
+    if not authnz.require_saml_role(_cred.metadata)['read_write']:
+        return jsonify({'error': 'Credential Metadata found Role does not have write permissions'}), 403
```

Comment:
Same as above. Would be good to make a decorator for this.

Reply:
I think you also need to apply this to `create_credential()`, and `map_service_credentials()`